mne/io/base.py

```diff
@@ -1525,15 +1525,17 @@ def plot(self, events=None, duration=10.0, start=0.0, n_channels=20,
              event_color='cyan', scalings=None, remove_dc=True, order=None,
              show_options=False, title=None, show=True, block=False,
              highpass=None, lowpass=None, filtorder=4, clipping=_RAW_CLIP_DEF,
-             show_first_samp=False, proj=True, group_by='type',
-             butterfly=False, decim='auto', noise_cov=None, event_id=None,
-             show_scrollbars=True, show_scalebars=True, verbose=None):
+             show_first_samp=False, show_real_time=False, proj=True,
+             group_by='type', butterfly=False, decim='auto', noise_cov=None,
+             event_id=None, show_scrollbars=True, show_scalebars=True,
+             verbose=None):
```

Comment:
I know that this parameter logically belongs next to `show_first_samp`, but usually we tack new parameters to the end (just before `verbose`, which should always be last). I'm not sure it makes a lot of sense in this function though, because we already have so many parameters and nobody should rely on using positional arguments. But we haven't enforced keyword args, so we should probably stick to our conventions.